code/go/0chain.net/chaincore/block/entity.go

```diff
@@ -1033,6 +1047,26 @@ func (b *Block) ComputeStateLocal(ctx context.Context, c Chainer) error {
 	return nil
 }
 
+func transactionNodeToEventTransaction(tr *transaction.Transaction, blockHash string) event.Transaction {
```

Comment:
Can we move this helper function to `event/trasnaction.go` ?

Reply:
No because doing that we are doing this in dependency tree
chiancore imports  smartcontract
smartcontract imports chaincore


This is not recommended